CONFIGURATION.md

```diff
@@ -0,0 +1,128 @@
+# Configuring NTPD-rs
+
+NTPD-rs is an implementation of NTP completely written in Rust, with a focus on exposing a minimal attack surface. The project is currently in an early stage, but if you want to try out NTPD-rs on a non-critical system, this guide provides the basic information needed to properly build and configure NTPD-rs.
+
+## Limitations
+
+The current implementation has several important limitations which it is important to be aware of. First of all, the current implementation is an NTP client only, and does not support acting as an NTP server.
```

Comment:
"important [..] important" reads a little awkwardly.

It's also odd to me that you have a bullet list (which is great, allows for easy scanning) but then mention this first item before/outside of the bullet list. I'd just make that part of the bullet list.

Reply:
Fixed the sentence, and made the section essentially a single bullet list